src/operator/keycloak.ts

```diff
@@ -70,7 +74,7 @@ interface RealmRole {
   name: string
 }
 
-const localEnv = cleanEnv({ KEYCLOAK_TOKEN_TTL, KEYCLOAK_TOKEN_OFFLINE_TTL, KEYCLOAK_TOKEN_OFFLINE_MAX_TTL_ENABLED })
+const localEnv = cleanEnv({ KC_SESSION_IDLE_TIMEOUT, KC_SESSION_MAX_LIFESPAN, KC_ACCESS_TOKEN_LIFESPAN, KC_ACCESS_TOKEN_LIFESPAN_FOR_IMPLICIT_FLOW, KC_OFFLINE_SESSION_MAX_LIFESPAN_ENABLED, KC_OFFLINE_SESSION_IDLE_TIMEOUT, KC_OFFLINE_SESSION_MAX_LIFESPAN })
```

Comment:
We usually call this just env, but its not a blocking issue
```suggestion
const env = cleanEnv({ KC_SESSION_IDLE_TIMEOUT, KC_SESSION_MAX_LIFESPAN, KC_ACCESS_TOKEN_LIFESPAN, KC_ACCESS_TOKEN_LIFESPAN_FOR_IMPLICIT_FLOW, KC_OFFLINE_SESSION_MAX_LIFESPAN_ENABLED, KC_OFFLINE_SESSION_IDLE_TIMEOUT, KC_OFFLINE_SESSION_MAX_LIFESPAN })
```

Reply:
If you apply this you also need to change it in the lines below.

Reply:
all the operators use this setup in tasks, I guess it should be changed when during the rework.